### PR TITLE
FUSETOOLS-2314 - FUSETOOLS-2187 - Display DataFormat parameters in

### PR DESCRIPTION
**Pull request by apupier**
Details Section

### CODE REVIEW

File: editor/plugins/org.fusesource.ide.camel.editor/src/org/fusesource/ide/camel/editor/properties/DetailsSection.java [hunk 1]
```diff
@@ -472,22 +461,25 @@ private IStatus validateMandatoryBehavior(final Parameter prop, IObservableList
...
 			return !group.equals(currentPropertyGroup);
 		}
 	}
 
 	private boolean isClassParamToHide(Parameter p) {
-		return CamelComponentUtils.isClassProperty(p) && AbstractCamelModelElement.NODE_KIND_ELEMENT.equalsIgnoreCase(p.getKind());
+		return CamelComponentUtils.isClassProperty(p)
+				&& AbstractCamelModelElement.NODE_KIND_ELEMENT.equalsIgnoreCase(p.getKind())
+				&& !CamelComponentUtils.isDataFormatProperty(p)
+				&& !CamelComponentUtils.isDescriptionProperty(p);
```
Comment (apupier) [inline editor/plugins/org.fusesource.ide.camel.editor/src/org/fusesource/ide/camel/editor/properties/DetailsSection.java]: should I also check for Expressions?

Comment (lhein) [inline editor/plugins/org.fusesource.ide.camel.editor/src/org/fusesource/ide/camel/editor/properties/DetailsSection.java]: you mean !CamelComponentUtils.isExpression(p) ?

Comment (apupier) [inline editor/plugins/org.fusesource.ide.camel.editor/src/org/fusesource/ide/camel/editor/properties/DetailsSection.java]: yes

Comment (apupier) [inline editor/plugins/org.fusesource.ide.camel.editor/src/org/fusesource/ide/camel/editor/properties/DetailsSection.java]: no need to check for expression, the kind is never Node, it is always "expression" in this case

**Comment by apupier** [comment]
this PR is ready for review

**Pull request merged**